laws/src/main/scala/cats/laws/discipline/MonoidKTests.scala

```diff
@@ -0,0 +1,28 @@
+package cats
+package laws
+package discipline
+
+import algebra.laws._
+import org.scalacheck.Prop._
+import org.scalacheck.Arbitrary
+
+object MonoidKTests {
+  def apply[F[_]: ArbitraryK, A: Arbitrary](implicit eqfa: Eq[F[A]]): MonoidKTests[F, A] =
+    new MonoidKTests[F, A] {
+      def EqFA = eqfa
+      def ArbA = implicitly[Arbitrary[A]]
+      def ArbF = implicitly[ArbitraryK[F]]
```

Comment:
This is a totally minor thing, but could you please make use the implicit parameter list for the above two implicit instances like you have for `eqfa`? This would make it consistent with the style convention that was decided upon in #27 (but that has yet to be properly documented).